AUTHORS

```diff
@@ -641,6 +641,7 @@ answer newbie questions, and generally made Django that much better:
     Michael S. Brown <michael@msbrown.net>
     Michael Hall <mhall1@ualberta.ca>
     Michael Josephson <http://www.sdjournal.com/>
+    MIchael Lissner <mike@free.law>
```

Comment:
Not sure if intended but you've got a capital `MI` here.

Reply:
Welp, I guess I can't spell my own name. Sigh.